VerilogCodingStyle.md

```diff
@@ -437,6 +441,26 @@ assign structure = '{src: src,
 Operators in a wrapped expression can be placed at either the end or the
 beginning of each line, but this must be done consistently within a file.
 
+Open syntax characters such as `{` or `(` that end one line of a multi-line
+expression should be terminated with close characters (`}`, `)`) on their
+own line. Examples:
```

Comment:
Does this need something explicit to allow the `zulu = ... delta` syntax from above?

Reply:
> Does this need something explicit to allow the `zulu = ... delta` syntax from above?

Do you mean line 429? That is theoretically explained by the "improves human readability" portion.

Reply:
I'm sorry, I was being slow. Yes, I did mean that line, and I thought this paragraph was disallowing that syntax. But the point is that this says "... `(` that end one line...", which doesn't apply to
```
enabled && (foo ||
            bar)
```

So all is well!